ConnectorRhino/ConnectorRhino/ConnectorRhinoShared/UI/MappingBindingsRhino.cs

```diff
@@ -58,8 +66,9 @@ public override MappingSelectionInfo GetSelectionInfo()
     }
     catch (Exception ex)
```

Comment:
I get we are throwing, but do we want to catch something more specific or wrap if !NotFatal()

Reply:
Interestingly... I see no warning for this on my IDE 🤔 

EDIT: Issue with my mac IDE... non related to this. I'm resolving this as it's been fixed